### ISSUE TITLE
Right side border of <select>/<option> is not showing on Windows Firefox 

### ISSUE THREAD

**Issue by sibinx7** [twbs/bootstrap#15990]
I have an issue with select boxs. When i place select input field inside a col-sm-3 it won't show right side border. It work on other grid sizes. Only Mozilla has problem all other browser everything work well.

update : col-sm-3, col-sm-5,col-sm-7,col-sm-9 and col-sm-11 have same issue

Bootstrap version : Bootstrap v3.2.0 (http://getbootstrap.com), Bootstrap v3.2.2 also have same issue
Please help me, thanks in advance 

![bootstrap-3](https://cloud.githubusercontent.com/assets/2009809/6476039/3392ac80-c237-11e4-8e97-db882c4265d6.jpg)

**Comment by cvrebert** [comment]
@sibinx7 What version of Windows?

**Comment by sibinx7** [comment]
Windows 8.1 Single Language 64 bit

**Comment by hnrch02** [comment]
Can you confirm that this is a problem with the latest version, v3.3.2?

**Comment by sibinx7** [comment]
I had changed version but still i have same same issues.

**Comment by sibinx7** [comment]
![bootstrap-3 2 2](https://cloud.githubusercontent.com/assets/2009809/6478974/6d115b92-c263-11e4-9636-592fcc8ed5b8.jpg)

**Comment by patrickhlauke** [comment]
Got a minimal test case/jsbin example?

**Comment by cvrebert** [comment]
I was unable to reproduce this with http://jsbin.com/hajoxa/2 on Win 8.1 Firefox 36 via Sauce Labs:
![0002screenshot](https://cloud.githubusercontent.com/assets/419884/6499852/f925386c-c2b6-11e4-88a1-3a190ccbdf3b.png)

**Comment by sibinx7** [comment]
Sir, can you check this link, i have many select boxs

http://jsbin.com/titaki


Sample code 
<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title>Select</title>
    <link rel="stylesheet" href="content/css/bootstrap.min.css"/>
</head>
<body>
<div class="container-fluid">
    <div class="row">
        <div class="col-xs-12 col-sm-3">
            <div class="form-group">
                <div class="row">
                    <div class="col-xs-12">

                        <select class="form-control " name="" id="">
                            <option value="">AUDI (2)</option>
                            <option value="">BMW</option>
                            <option value="">FIAT</option>
                            <option value="">Volvo</option>
                        </select>

                    </div>
                </div>
            </div>
        </div>
        <div class="col-xs-12 col-sm-3">
            <div class="form-group">
                <div class="row">
                    <div class="col-xs-12">

                        <select class="form-control " name="" id="">
                            <option value="">AUDI (2)</option>
                            <option value="">BMW</option>
                            <option value="">FIAT</option>
                            <option value="">Volvo</option>
                        </select>

                    </div>
                </div>
            </div>
        </div>
        <div class="col-xs-12 col-sm-3">
            <div class="form-group">
                <div class="row">
                    <div class="col-xs-12">

                        <select class="form-control " name="" id="">
                            <option value="">AUDI (2)</option>
                            <option value="">BMW</option>
                            <option value="">FIAT</option>
                            <option value="">Volvo</option>
                        </select>

                    </div>
                </div>
            </div>
        </div>
    </div>
</div>
</body>
</html>

**Comment by twbs-lmvtfy** [comment]
Hi @sibinx7!

You appear to have posted a live example (http://jsbin.com/titaki/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 17, column 68: Bad value ````  ```` for attribute ```` id ```` on element ```` select ````: An ID must not be the empty string.
* line 33, column 68: Duplicate ID ````  ````.
* line 33, column 68: Bad value ````  ```` for attribute ```` id ```` on element ```` select ````: An ID must not be the empty string.
* line 49, column 68: Duplicate ID ````  ````.
* line 49, column 68: Bad value ````  ```` for attribute ```` id ```` on element ```` select ````: An ID must not be the empty string.
* line 65, column 68: Duplicate ID ````  ````.
* line 65, column 68: Bad value ````  ```` for attribute ```` id ```` on element ```` select ````: An ID must not be the empty string.
* line 81, column 68: Duplicate ID ````  ````.
* line 81, column 68: Bad value ````  ```` for attribute ```` id ```` on element ```` select ````: An ID must not be the empty string.
* line 97, column 68: Duplicate ID ````  ````.
* line 97, column 68: Bad value ````  ```` for attribute ```` id ```` on element ```` select ````: An ID must not be the empty string.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Still no luck reproducing it with your bin:
![shot1](https://cloud.githubusercontent.com/assets/419884/6500622/e86b6c92-c2c2-11e4-8669-902478dbfcf6.png)

**Comment by sibinx7** [comment]
Screen resolution 1366 x 768, Issue happened only in some resolution

![bootstrap-3 2 2-select](https://cloud.githubusercontent.com/assets/2009809/6500806/83d710e6-c336-11e4-9857-a1e7a69c5827.jpg)

**Comment by patrickhlauke** [comment]
I just set my Win8.1 machine to 1360x768, but also still can't reproduce the problem I'm afraid

![fx1360x768](https://cloud.githubusercontent.com/assets/895831/6502669/74c7e56c-c31d-11e4-85e5-75d8dc550d86.png)

**Comment by sibinx7** [comment]
Sir

Please check 1230 X 768 , 1366 X 768 , 1290 X 768 , 1346 X 768 etc.. Please check on these resolution.

My default resolution is 1366 X 768

**Comment by patrickhlauke** [comment]
ah, i had to force my monitor to accept 1366x768, and now i can see the issue in Firefox. however, as soon as i move the Firefox window slightly, the border appears. This leads me to believe it's a bug in Firefox's rendering engine and the slightly unorthodox resolution used (a more standard one would be 1360x768. Other browsers are also fine using 1366x768, so it's definitely something broken with Firefox itself, which unfortunately is outside of our control.

![fx-1366x768](https://cloud.githubusercontent.com/assets/895831/6504385/91e3c6aa-c32e-11e4-8cf0-fb1188719e18.png)

**Comment by cvrebert** [comment]
Upstream bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1139853

**Comment by cvrebert** [comment]
Belatedly added this to [the Wall](http://getbootstrap.com/browser-bugs) in #18448.

**Comment by cvrebert** [comment]
Upstream bug was closed as a duplicate of an existing bug.
New relevant bug: https://bugzilla.mozilla.org/show_bug.cgi?id=924068
